application/tests/org.openjdk.jmc.flightrecorder.ext.jfx.test/src/test/java/org/openjdk/jmc/flightrecorder/ext/jfx/test/JfxPulseDurationRuleTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Don't forget to update the RedHat year as well.

Reply:
Hm, shouldn't one not just update the Oracle copyright year and that of one's own company, if applicable?

Reply:
fixed.